### PR TITLE
Enforce min and max length with strings

### PR DESCRIPTION
**Pull request by ganymedes01**

### CODE REVIEW

**Review by jasujm — APPROVED** [review]

File: packages/oats-runtime/src/make.ts [hunk 1]
```diff
@@ -172,6 +174,12 @@ export function makeString(
     if (typeof value !== 'string') {
       return getErrorWithValueMsg('expected a string', value);
     }
+    if (minLength != null && value.length < minLength) {
+      return getErrorWithValueMsg('expected a string with length greater than ' + minLength, value);
```
Comment (sakari) [inline packages/oats-runtime/src/make.ts:178]:
```suggestion
      return getErrorWithValueMsg('expected a string with length of at least ' + minLength, value);
```

File: packages/oats-runtime/src/make.ts [hunk 2]
```diff
@@ -172,6 +174,12 @@ export function makeString(
...
+    }
+    if (maxLength != null && value.length > maxLength) {
+      return getErrorWithValueMsg('expected a string with length smaller than ' + maxLength, value);
```
Comment (sakari) [inline packages/oats-runtime/src/make.ts:181]: ditto .length === maxLength is allowed

Comment (ganymedes01) [reply packages/oats-runtime/src/make.ts:181]: is it? it wasnt clear to me from the specs if it was inclusive or not

Comment (sakari) [reply packages/oats-runtime/src/make.ts:181]: I would kinda read this https://cswr.github.io/JsonSchema/spec/basic_types/ as meaning that

**Pull request merged**